### ISSUE TITLE
german translations

### ISSUE THREAD

**Issue by typxxi** [dckiller51/bodymiscale#17]
### **UPDATE 05: 21 th of JUNE: 'background picture version' ** 
Okay, you had developed it further regarding scale number and background image or so, but here is bug I explain in point 2.

![image](https://user-images.githubusercontent.com/20662430/122691408-c7a97700-d22f-11eb-8b2a-262d2269eec7.png)
1. upper right seems to be fixed right
2. upper left has a new bug as it said after installation of the card behind the scale "unknown" I guess we need to call that "STATUS: unbekannt" and you therefore should change the "OK" value to "Messung OK". That way both messages are accurate for its purpose
3. I suggest for the card configuration dialog to change the label of the picture from "Bild (Optionall)" to "Hintergrundbild - optional" or "pic" to "background pic" cause it is about a background pic. ATTENTION: copy the whole term, cause Optional had changed too cause it need lowercase and no () 
4. next line is "Modell 181B oder 181D - erforderlich"  you should copy as a whole cause I always used a " - " as separtor and wrote "erforderlich" in  lower case as needed.
5. And then there is still a lot of details that regarding translation needed still to be fixed and I had been discussing already earlier.
6. Be aware to change the built number in case you update it again cause that is the first part where I can see if those can or will work out. 

### **UPDATE 04: 20 th of JUNE: 6 pm - to do's on the new version 0620 ** 

I guess this will be a shorter list of what is left to be done

1. OLD: Search for STATUS WAAGE and replace that whole line by what I had written in the list
"show_state": "Status anzeigen (Symbole links oben)"
I added the description to make clear what it is for "(icon/s upper left)"
ATTENTION: If you agree that you will end up with max 1 symbol shown at a time for the scale states then you have to take the following line cause that would mean a singular icon and therefore "Symbol" instead of Plural "Symbole" right now where you show scale and warning sign together. If you will endup with 1 sign at a time then this would be the right line: 
"show_state": "Status anzeigen (Symbol links oben)"

2. OLD: Please do a search and replace for "Nicht verfügbar" by "nicht verfügbar" cause that appears quite often.

3. OLD: search for "kein" or "none" and replace the word "kein" bei "keine" caue no and none are different, same for kein / keine

4. OLD: Search for "Körpger..." and delete that "g" which is a typo.  

5. NEW: Now search for "Körperwert und" and replace "Körperwert" by the infiltrated german word "Power Button"

6. OLD: Try to add your "Body Mi Scale Card" to the "add card dialog" cause it is missing even though not important right now, but looks a lot better for sure 
![image](https://user-images.githubusercontent.com/20662430/122681444-9eb7c080-d1f4-11eb-97a6-e34141f3aa86.png)

7. OLD: just a kind reminder that you wanted to adjust the kg decimals to just 2 for both / all weights cause currently they are different from 0 to 2 and that does not look very well.

8. OLD: metabolism figure should be formated with a dot for the usual 1.000 + kcal cause it looks a lot nicer and is not a year.

Finally try to make the folloing fit in 2 lines as the pic i created shows and please replace "Button" by "Power Button"
If possible do the same for the warning error in case we get 2 messages that do not fit in 1 line cause they are so much easier to read and understand then a broken line. 
![image](https://user-images.githubusercontent.com/20662430/122681056-d02f8c80-d1f2-11eb-9aaf-1388d2340f89.png)

Final point I hope: the details shown by a press of the power button  should only display what can be available on that particular model. i do not need a 181B series of labels where the data are described as not available. Hide the unavailable pieces if possible.

So far, so good - except I can not check those values I do not have so maybe you can send me a screenshot when you use that development template and get the german UI. I mean just the body details you can show by the power button.

### **UPDATE 04: 20 th of JUNE: 5 pm - can not find any changes right now ** 

I just checked it and not a lot has changed or maybe I got the wrong file or do I have to completely reboot the pi 4?

![image](https://user-images.githubusercontent.com/20662430/122677772-18937e00-d1e4-11eb-8333-7ce3fae1a904.png)
all the "Nicht verfügbar" still there instead of "nicht ..." and "kein" in the top or status area instead of "keine" in the sense of "no warning messages" . So I assume my update by copy the raw from github into the js file did not really work out with just a docker stop homeassistant and docker restart homeassistant.

### **UPDATE 03: 20 th of JUNE: missing translations proven and corrected** 

Ok, that was an easy on for me and I hope for you too cause it is only about 7 terms with minor changes.

ATTENTION: 
#1 I simply translated and did not look for the key question if it might fit or not cause that has to be done in the next dev version but I guess you can see in most cases german words or terms are longer measured by letters.
#2 You should change a variable or label name I guess cause yours is wrong and might cause other issues.
"Lack-exerscise": the last should be "exercise" without that "s" before the "cise"
#3 tiny topic: search for Geschlecht and then you should find "männl" which needs to be "männl." - the dot is missing which allows us to shorten it from mänlich to männl. but only with a dot at the end.

**_Terms to exchange have NO LEADING'#'_**
```
# "Skinny": "schlank"
# "Balanced-skinny": "ausgeglichen schlank"
# "Skinny-muscular": "muskulös schlank"
# "Balanced": "ausgewogen"
# "Balanced-muscular": "ausgewogen muskulös"
"Lack-exerscise": "Bewegungsmangel"
# "Thick-set": "stämmig"
# "Obese": "fettleibig"
"Overweight": "Übergewicht"
# "Underweight": "Untergewicht"
"Normal or Healthy Weight": "Normal - gesundes Gewicht"
"Slight overweight": "leichtes Übergewicht"
"Moderate obesity": "moderate Fettleibigkeit"
"Severe obesity": "schwere Fettleibigkeit"
"Massive obesity": "massive Fettleibigkeit"
```

### **UPDATE 02: 19 th of JUNE: some translation improvements** 

Meanwhile I came across a few points that do not really work out regarding translation. 
To avoid unnecessary double work you can search for the word 'IMPROVED'

**ATTENTION:** above that IMPROVED you will find a changed line / term cause I had found typos and other smaller points or shortened terms like 'stoffwechselbedingtes Körperalter' in the hope that it fits in 1 line - with 2 lines it did not look good.

**FINAL 3 POINTs** 
the label "BMI-Label" of the lower / hidden section with the details is NOT covered at all in the translation file you had sent to me and also not its values as can be seen in this example screenshot "Leichtes Übergewicht" which has to be "leichtes" cause I was wondering why I had written with a capital letter.
And there is another phrase missing in that file cause I am wondering myself about the spelling of those translations and why I had done it that way. So I do  a search through the translation file for "Nicht verfügbar" with a capital "N" in the beginning. It is not in that file. So there might be more words missing a translation. here the final 3 tasks to not forget.

#1 Please change the term "BMI-Label" in the german version into **"BMI Klassifikation"** 
#2 Also change at least its value "Leichtes" into "leichtes" and send me the other german values to crosscheck those too
#3 Finally do a search and replace for "Nicht verfügbar" with "nicht verfügbar".  

![image](https://user-images.githubusercontent.com/20662430/122653657-52ae4280-d146-11eb-8c1e-542a416ed691.png)

### **UPDATE: 01 16th of JUNE: DONE** 
First task: please change and release the current one in 2 major points cause they disturb germans for sure cause now it shows 'average 200 cm' due to translation error and 'Nice: male' cause gender has been mis translated.

### **FIRST STATUS: 15 th of JUNE**
Please read my translation comments directly below the german line term just to think about if maybe some english terms should be adopted or expanded too. 
Then THANKS A LOT for the body-miscale-dev.js piece you created for me cause that way I can play around without damaging anything except my own config. 

To get your translation file back you simply have to delete all the empty lines and those that begin with # 

As said on the home assistant forum here is a long version of my thoughts to explain why some points felt a bit misleading / to short in english. 

Regarding organisational structure of my comments. For a better comparison I first had added the english original quote as a comment and sometimes also the first Body Scale App results to get a history what I changed from where to what.
The first translations you had send as german version are partly false, partly not understandable or misleading, but 60% were good, but as you had explained later that those german terms were from the app.

Therefore I hope you take the time to read these comments cause some might usefull to adopt to the english version too and most likely french, but it will of cause be a struggle regarding space if you focus on Smartphone developement.
 
```
{
    "common": {
      "name": "BodyMiScale Karte",
      # First question or point: I changed to upper letter whereever needed like in name. BodyMiScale is far better cause you can idenitfy its meaning a lot easier then an endless row of letters. I translate everything into german where younger people are used or more familiar with the english card which translated means Karte like a card in a card game. BodyMiScale itself is a productname so I do not tranlate scale into "Waage"

      "description": "Die BodyMiScale Karte kann Ihnen hren gewichtsmäßigen Körperstatus anzeigen.",
      # IMPROVED "description": "Mit der BodyMiScale Karte können Sie Ihren gewichtsmäßigen Körperstatus anzeigen lassen.",
      # name:"Bodymiscale Card",description:"The bodymiscale card allows you to display your body score."
      # here i added body status but "weightwise" which are just weight related data not body status regarding blood pressure.

      "not_available": "BodyMiScale ist momentan nicht verfügbar",
      #if you meant to say that the scale is not available I would call it "Ihre BodyMiScale Waage ist momentan nicht verfügbar" or if you meant the service itself then I would call it "Der BodyMiScale Dienst ist momentant nicht verfügbar" and I added "actually not available" "momentan nicht verfügbar"

      "toggle_power": "Weitere Details wie BMI kCal anzeigen / ausblenden"
      # Punktzahl is a kind of score you achieved in an examin or decathlon competition. I guess you mean "showing further details"      
    },
    "state": {
      "ok": "Messung: OK",
      # IMPROVED "ok": "OK",
      "problem": "Problem",
      "none": "keine",
      # IMPROVED "none": "kein",
      # none does not mean no, so I guess "kein" Problem would make more sense "none": "Nein",
      "weight unavailable": "Gewichts Messung nicht verfügbar",
      # we would say that the weight measurement is not available , not just weight only
      
      "impedance unavailable": "Bioelektrische Impedanz Messung (Körperzusammensetzung) nicht verfügbar",
      # to be honest I die not know what that means in the context of weight, I only knew it from springs and physics. And I guess millions of germans do not know that by just that single latinword Impedanz therefore if someone wants to know more about that he has to search for "Bioelektrische Impedanz (Körperzusammensetzung) Messung" or bioelectrical impedance measurement (body composition) as a kind of body analysis 
      
      "weight unavailable, impedance unavailable": "Gewichts und bioelektrische Impedanz Messung (Körperzusammensetzung) nicht verfügbar."
    },
    "attributes": {
      "weight: ": "Gewicht: ",
      "impedance: ": "Zusammensetzung: ",
      "height: ": "Körpergröße",
      # IMPROVED "height: ": "Körpgergröße",
      # you had here "Schnitt" which means average so most likely a copy paste error 
      
      "age: ": "Alter: ",
      "gender: ": "Geschlecht: "
      # you had here "Nett" which means nice ... but it should be Geschlecht
    },
    "attributes_value": {
      "male": "männl.",
      # you had here "Mann" which is a noun but male is an adjective and therefore I guess you meant the gender as adjective and not a question of "are you a man or woman question". I have shortened it from männlich to make it fit.
      
      "female": "weibl.",
      # you had "Frauen" = women, where it should be a single woman or "Frau", but as adjective "weiblich", shortened to weibl."
      
      "unavailable kg": "Gewichtsmessung momentan nicht verfügbar",
      # you had here "nicht verfügbar" which would mean not available but I guess it is about "weight measurement is not available".

      "unavailable ohm": "Bio Impedanzmessung momentan nicht verfügbar"
      # attention: here is a bug in the english word if it meant ohm, the measurement of resistance of electrical conductors. I guess you mean "Bio Impedance measurement is currently unavailable"
    },
    "body": {
      "water": "Wasser",
      "visceral_fat": "Bauchfett",
      "body_fat": "Körperfett",
      "bmi": "BMI",
      "muscle_mass": "Muskelmasse",
      "protein": "Protein",
      "basal_metabolism": "Grundumsatz",
      "bone_mass": "Knochenmasse",
      "metabolic_age": "stoffwechselbedingtes Alter",
      # IMPROVED "metabolic_age": "stoffwechselbedingtes Körperalter",
      # too complex to translate but it means "metabolic related body age" 

      "ideal": "Idealgewicht",
      # I guess this means "ideal weight" if I remember right and therefore we have term, a compound word as usual: Idealgewicht
      
      "body_type": "Körperbau"
      # the former translation "Körpertyp" is not from a native speaker ... it means how the body is build or body build.
      
    },
    "body_value": {
      "Skinny": "schlank",
      # I replaced all those skinny = dünn by "schlank" cause that is the typical description, a compliment where "dünn" could mean a critique 

      "Balanced-skinny": "ausgeglichen schlank",
      "Skinny-muscular": "muskulös schlank",
      "Balanced": "ausgewogen",
      "Balanced-muscular": "ausgewogen muskulös",
      "Lack-exerscise": "Bewegungsmangel",
      "Thick-set": "stämmig",
      "Obese": "fettleibig",
      "Overweight": "übergewichtig"
    },
    "unit": {
      " years": " Jahre"
      # is the leading space intentionally there or just a typo ?
      
    },
    "error": {
      "missing_entity": "Bitte definieren Sie einen Waagen-Namen Sensor in der Konfiguration.",
      # I added some words cause define an entity or Entität sounds weird for germans that mostly do not know what an Entität is. So I called it "please define a scale-name sensor in the configuration."
      # "missing_entity": "Please define an entity.",
      #
      
      "missing_enttity_bodymiscale": "Bitte definieren Sie den Waagen Sensor in der Konfiguration.",
      # I added some words cause define an entity or Entität sounds weird for germans that mostly do not know what an Entität is. So I called it "please define the scale sensor in the configuration." 
      # "missing_enttity_bodymiscale": "Please define a bodymiscale entity.",
          
      "missing_model": "Bitte definieren Sie ein gültiges Modell der Waage wie miscale, 181D, 181B in der Konfiguration."
      # ATTENTION: this one is complex cause there has been an error, but here first of all your german and english one and then my explanation.
      # "missing_model": "Bitte definieren Sie ein gültiges Maßstabsmodell."
      # "missing_model": "Please define a valid scale model."
      # I simply have no clue cause the german model is modell RC related like a scale plane model in 1:10 of a Mirage fighter cause the chinese have mistranslated the device scale into the attribute scale like. I think this english sentence means that you have to choose the right scale in terms of right model like 181D 
      # my translation means: please define only a valid model of the supported scales like miscale, 181D, 181B in the configuration"

    },
    "editor": {
      "entity": "Auswahl der Waage / Person erforderlich"
      # UPDATE: "entity": "Waage / Sensor erforderlich",
      # I guess it means a sensor is neccessary
      # "entity": "Entity (erforderlich)",
      
      "show_name": "Namen anzeigen",
      # UPDATE "show_name": "Name anzeigen", 

      "show_name_aria_label_on": "Namensanzeige einschalten",
      "show_name_aria_label_off": "Namesanzeige ausschalten",

      "show_state": "Status anzeigen (Symbole links oben)",
      # I added the description to make clear what it is for "(icons upper left)"

      "show_state_aria_label_on": "Statusanzeige einschalten",
      "show_state_aria_label_off": "Statusanzeige ausschalten",

      "show_attributes": "Basis Daten einblenden (rechts oben)",
      # I changed this cause the Attribut term does not fit at all. I called it "show basic data (upper right)"
      "show_attributes_aria_label_on": "Basis Daten einblenden (rechts oben) einschalten",
      "show_attributes_aria_label_off": "Basis Daten einblenden (rechts oben) ausschalten",
      # I have not seen where these labels are used so hope that fits

      "show_body": "Körperwertanzeige anbieten (untere Hälfte)",
      # I call this an offer to display body status details cause you have to press the button to see any change. They are not instantly there. I also added where that change will be happening (lower half)
      # it was "show_body": "Körperwert anzeigen" and does not reflect what will happen cause instantly nothing will change except the toggle switch will work from there on,
      "show_body_aria_label_on": "Körperwertanzeige einschalten",
      "show_body_aria_label_off": "Körperwertanzeige ausschalten",

      "show_buttons": "Schaltfläche anzeigen",
      # I do not now know what this is for but I guess it is the button to change a person in a multipurpose environment. 
      "show_buttons_aria_label_on": "Schaltfläche anzeigen einschalten",
      "show_buttons_aria_label_off": "Schaltfläche anzeigen ausschalten",

      "show_toolbar": "Symbolleiste (Körperwert und Schaltfläche) anzeigen",
      # I added that the toolbar consists out of (body aria and buttons
      "show_toolbar_aria_label_on": "Symbolleiste anzeigen einschalten",
      "show_toolbar_aria_label_off": "Symbolleiste anzeigen ausschalten",
      "code_only_note": "Bitte beachten: Die Aktionen und Auswertungsoptionen sind nur im Code Editor verfügbar."
      "code_only_note": "Note: Setting actions and stats options are available exclusively using Code Editor."
    }
  }
```

**Comment by dckiller51** [comment]
**UPDATE 04 : 20 JUIN : 18h00 - à faire sur la nouvelle version 0620 **
1) I keep in mind to display the second icon only in case of problem.
2) I can't find a line with an "N", I only find "n"
3) made
4) made
5) The buttons are those on the right in the toolbar. I keep hoping to assign one button per user. Tell me if we are in phase.
6) I think we don't have the hand. I looked at other personalized card the name remains card configuration
7) The two decimals are to be updated in the custom-components bodymiscale. For the moment I have not found the right formula
8) Do you think this could be confused? 
I don't understand why the number doesn't take the profile parameter
1,000 (EN) 1 000 (FR) 1.000 (DE) 

For the 181B display you have the option to add model: 181B or model: 181D
I must find a way to have a selection of model in order to have it directly on the configuration of the card

**Comment by typxxi** [comment]
> **UPDATE 04 : 20 JUIN : 18h00 - à faire sur la nouvelle version 0620 **
> 
> 1. I keep in mind to display the second icon only in case of problem.
> 2. I can't find a line with an "N", I only find "n"
> 3. made
> 4. made
> 5. The buttons are those on the right in the toolbar. I keep hoping to assign one button per user. Tell me if we are in phase.
> 6. I think we don't have the hand. I looked at other personalized card the name remains card configuration
> 7. The two decimals are to be updated in the custom-components bodymiscale. For the moment I have not found the right formula
> 8. Do you think this could be confused?
>    I don't understand why the number doesn't take the profile parameter
>    1,000 (EN) 1 000 (FR) 1.000 (DE)
> 
> For the 181B display you have the option to add model: 181B or model: 181D
> I must find a way to have a selection of model in order to have it directly on the configuration of the card


1. if you have to show the 2nd symbol then do not forget to add behind to the scale icon this 
"State: error" or german "STATUS: Fehler" which would be read as 'Scale state: error'

2. but it is there as you can see in that screenshot 
![image](https://user-images.githubusercontent.com/20662430/122761088-24427b80-d29c-11eb-8092-a12f5bcb98bb.png)

5. You talk about buttons in plural but right now I could only see 1 (due to just 1 sensor in my yaml file I guess). 
I can only see 1 switch with a "U" I guess. If you mean that there could be "1 or a set of switches" to switch between users we should make that pretty clear in the configuration and give it a name like "Scale account switch(es)"  cause for me these so called entities are simply combinations of scale and user cause you might have 2 scales in bath and bedroom and so on. I personally would call these scale accounts. How will  multiple users work - by adding multiple user accounts as sensors into the yaml file ?  

6. I am not sure what you meant but I guess you meant that you can not change the title bar here
![image](https://user-images.githubusercontent.com/20662430/122763238-6371cc00-d29e-11eb-9c65-f06799046810.png)
You mean that this is not be possible for personal cards I guess, while others in general can do that ? 
AND: I checked it too for mini graph and custom bar card and another 2 where none had or chould have changed it. 
Sad but true, would be very helpfull especiall regarding support. 

7. Regarding decimals and consistency you had told me that you did not know it right now but I put it on my list that it should not be forgotten. I had already made a new list which separates this whole thing into sections like tranlsation, features, structure and so on to make things clear were we are. I just keep track about the things and what is not available yet can become available tomorrow if it is usefull enough and you have access / techniques to change that.

8. I am not the expert but as I can see even france has its own "deal" treatment of "thousand" separators. Usually we had gotten and later adopted french habits during the napoleonic occupation but somehow we had gotten the "." as thousand separator and not a " ". I like a that "." a lot more than the british american "," which can drive me crazy. 

But due to the fact that the underlaying "HA" is capable to deal with it on a user level and not on an installation level it should work out of the box but it does not in multiple occasions cause it is running in US mode I guess.

If I ask you a lot others can ask that too and it is easier to have a list of things in mind that are not working as hoped cause I guess that you would prefer then french "1 999 kcal" over the crazy german "1.999 kcal" even thought the amount stays the same may be a tiny french champaign compared to a big german beer but the energy amount will stay the same ... 
![image](https://user-images.githubusercontent.com/20662430/122764790-1c84d600-d2a0-11eb-8bf5-f2088d321950.png)


Regardless all these points I had been working on an updated list for this github page which I will check once again. If things might appear again or differently be aware that this new list should be a kind of overall status report cause we are working on the card itself in general and the translation topics in parallel. But the new one will have a stronger separation into translations, general stuff and things to not forget like decimals / thousand separators and so on. 

I guess you might call on HA for "CARD EXPERTISE NEEDED: how to adjust / handle with separators " or so cause someone might know, same for the question if customizeable cards can not get its own name in the title bar and why not. 
It does not make sense cause the code itself begins with a clear statement of responsibility:

```
type: 'custom:body-miscale-dev'
entity: bodymiscale.wolf
image: default
model: 181D
show_toolbar: false
```
Therefore that "type" should allow to change that title bar even to "custom: Lovealce Body Mi Scale (0621) Card Configuration" 

Once again, this is my reply to your thoughts above while I had been working on the next "stage" since I was able to get it working and therefore called by me the 0620 late night edition. 

Please give the new version a version number 0620-hh  where hh stands for the hour of release to differentiate between those cause right now I was not sure how to call that and went for the 0620 late night edition cause before I could not get it to work.
The first edition in the noon was easy to get that running but that 18:00 one was a nightmare cause I never knew what I had gotten or not cause I had no spot to check. 

Therefore you should add that release number additionally at the end of the last configuration line here after the word verfügbar in brackets like "... verfügbar [0621-18]." or "... available [0621-18]."
 
![image](https://user-images.githubusercontent.com/20662430/122767432-f3197980-d2a2-11eb-99f2-01f8fab75a33.png)
 
Finally: do not be astonished with the next comment cause for example I have already "developed" this last sentence further cause in that current shape it is not as good as it should be but I will explain it there again and did not want to mix my reply to your status up with my upcoming comment.

**Comment by typxxi** [comment]
I. EVALUATION OF LATE NIGHT BUILT 0620 
=============================
First note goes to the built number: 0620-hh should be the new sheme where hh are representing the hour  of release cause I had yesterday only gotten 0620s

I will start with pure translation points except that I wanted to start from top to bottom and therefore begin with a non translation issue but a more general one which you had already answered and I added your reply as state: not possible yet.

#1
card configuration title should be 
`"Lovelace body miscale Card Configuration"`

For me it should be the spot #1 for the BUILT number like cause I was struggling a lot what code my card was relying on which is very frustrating to not be able to get behind which code is running except by differentiating that due to its translation errors  
`"Lovelace body miscale [0620-hh] Card Configuration"`  
STATE: post poned cause right now not possible

#2
A new error or untranslated term had appeared behind the scale icon "unknown" 
![image](https://user-images.githubusercontent.com/20662430/122768868-4f30cd80-d2a4-11eb-9656-140cc5470553.png)

I guess we need to call that "STATUS: unbekannt"  (means "STATE: unknown")
Therefore the previous "OK" value has to change too into "MESSUNG: OK" (means "MEASUREMENT: OK"). 
That way both messages are accurate for its purpose, first for scale in general, second is about the measurement results.

Search for 
`"OK"`
and replace the whole german term by this
`"MESSUNG: OK"`

And regarding "unknown" you have to implement that on your own as "state: unknown" and for german "STATUS: unbekannt"

#3
Search for 
`"Auswahl der Waage / Person erforderlich"`
and replace the whole german term by this
`"Bitte ein Waagen Konto auswählen [erforderlich] !"`

After being unsatisfied with the previoius term (which is right but feels weird to read and explain) we now could go the technical way which would mean a "Sensor - user name" combination or much easier "Please choose your scale account [mandatory] !"

You have a scale and even if you are alone the system it can only work if you have created at least 1 account in that bodymiscale.yaml file - can you tell me that a second user has to be added in that file too ?

#4 
Search for 
`"Bild (Optional)"`
and replace the whole german term by this
`"Hintergrundbild [optional]"`

For me it is not about "pic" but about a "background pic" cause the figures will lay in front that (background image). 

#5 
Search for 
`"Detaillierte Messergebnisse anbieten" `
and replace the whole german term by this
`"Detaillierte Messergebnisse anbieten (untere Hälfte - per Power Button einblenden)"` 

This means "Offer detailed measurement results (lower part - press power button to show)"

As said before it would be nice if there is a way to add a carriage return line feed / new line after "... anbieten"
`"Detaillierte Messergebnisse anbieten`
`(untere Hälfte - per Power Button einblenden)"`
I had seen that before but I can not tell  you where.

#6
Search for 
`"Schaltfläche anzeigen" `
and replace the whole german term by this
`"Schalter (U) anzeigen (rechts unten)"`

Previous "Schaltfläche" means a (bigger) switch area like 'cancel' or so, but what you mean is that U spot / button which is called in german "Schalter" / button and where I added (U) cause it looks like an "U" in a white circle switch.

#7
Search for 
`"Symbolleiste"`
and replace the whole german term by this
`"Symbolleiste anzeigen (Power Button und Schalter)"`

I had to change the terms inside to "Power Button" (the translation "Einschaltknopf" sounds weird in that context cause it would mean to turn the whole scale on / off which it does not do). 

Therefore I still suggest to use any different kind of button instead that particular turn on / off button which will cause questions.

I still recommand an icon that represents to hide / unfold something like a triangle or twister ᐅ to press for more details and if that had been pressed I suggest to replace that previous ᐅ with the opposite twister / triangle like this ▼ cause this switch has a toggling meaning to show or hide details. 
A power button on the other side would always target on a device as a whole like turn on / off your complete HiFi not a particular feature on that stereo. 

#8
Search for 
`"Bitte beachten ...."`
and replace the whole german term by this
`"ACHTUNG: Weitere Optionen sind nur im Code Editor verfügbar. [0620-hh]"`

which means
`"ATTENTION: Further options are only available in the code editor. [0620-hh]"`

QUESTION: I have deleted the word "actions" cause I was not sure if there are any actions available or not and if actions have to be mentinoned there cause usually it will cause questions like "What actions ?" 

I'd prefer the short and simple one cause that would work better on tablet or smartphones too and would not lead to ask what other "actions" are available. 

II. GENERAL CONFIGURATION
=============================

This chapter is beyond translations, more integral questions in general like in which order why option should appear or even been hidden cause it makes no sense to offer switches that will not be shown if the general symbol bar is off, but that explanation will follow below.

Just an example that you first need a symbol bar that is turned on to then make those other switches available.

**I.**
You need a new first line as explanation of the general behaviour
BIG LETTERS: CHANGES WILL ONLY APPEAR AFTER BEING SAVED.

Why is that important ? 
I had struggled with it but only in certain cases. While in configuration you can press any of those 6 switches but nothing will change before you will have saved it. But on the right half iof the configuration dialog you have a preview and for example that power button in the preview will be working regardless how the switches are switched or not.  

**II.**
Please move that "model number" input line above that "Background image" line for the simple reason that all mandatory inputs should get the highest amount of attention in the top section and should be selected first followed by the less important optional ones like that Background image.

**III.**
I would love to see a better structure in the configuration especially regarding these in total 6 switches.

![image](https://user-images.githubusercontent.com/20662430/122775510-7ee2d400-d2aa-11eb-8f16-0cc8f7dc1ef1.png)

Therefore can you please add an empty line (or separator line out of ----) above the first switch which is followed by this line
```"Options for card header" = "Kartenkopf Optionen"```

**IV.** 
After these first 3 switches I would love to see 
an empty line
followed by this additional  
`"Further card options (measurement details / switches)" or in german "Weitere Kartenoptionen (Messung Details / Schalter)"`
Yes, both german versions are this time shorter !

It will make it a lot easier in the beginning to understand where the targets of the upper half are and what each switch is for while we both do it know now blindly due to daily training but the average user will struggle which swtich is what for (therefore I had pointed out where those things are located).

**V.** 
Right now the switches 4, 5, 6 are not in the RIGHT and an INTUITIVE order !
`4 show button for more measurement details `
`5 show switch`
`6 show symbol bar`

But that LAST SWITCH "show symbol bar" has to be the first choice in the lower section cause **without showing the symbol bar** switches 4 and 5 are useless to click on and therefore should stay hidden / unavailable, right ?

Only once "symbol bar" is activated the other 2 switches should appear cause only then they can work.
Without icon bar enabled the "show button" and "show switch" are running into the emptineess as we called it here.
`4 Show symbol bar`
`5 Offer button for more measurement details` 
`6 Show switch`

I chose "offer button for more measurement details" cause you only offer a button which has to be pressed to show further measurement details. This is not a general switch to always show the measurement details. "show" instead "offer" will be misunderstood cause if you would activate that switch "show button for more measurement details" people will mostly only read and expect "show more measurement details"

**VI.**
The **"Switch (U)"** should get a title like **"Show scale account selector(s)"** and that would finally mean to replace "show switch" by "Show scale account selector(s)" ? 
ATTENTION: if you already have done the translation tasks mentioned above in the first section then you will have to do this

Search for 
`"Schalter (U) anzeigen (rechts unten)"` 
and replace the whole german term by this
`"Den / Die Waagekonten Auswahl Schalter (U) anzeigen (rechts unten)"`

**VII.**
Regarding the **model number input** I was not happy too and I am not convinced about that way you had implemented cause in reality it is - at least right now - just a choice between 2 scales and nothing more. Therefore: 

**Isn't a switch a better solution to chose between 2 models where '181D' should be the default setting?**
I suggest as title "**Do you own model 181B incl. impedance measurement?**"

By default it is turned of and therefore it will run as a model 181D user. 
Everyone with a better model does not loose much with such basic setting and can switch easily to the better 181B to get more measurement details.
If the "poor" 181D owner turns on this switch for the 181B model he will get a lot more label but no values and learns what the differences are between those similar looking scales - as I had learned it too from you and your code / data. 

That typing solution is not ideal and it would never be my first choice even though you read the initial value out of the yaml file and you might write that value back too. Such switch would work at least right now a lot better and looks nicer too for sure.
 
**AND:** a **switch does not need the additional "mandatory" term** too cause with a switch the card would start by default as 181D.

III. POINTS TO NOT FORGET
=============================

**1. ALL ABOUT FIGURES AND FORMATS**
* usefull, consistent decimals like 2 decimals for weight
* metabolism kcal figure with national / domestic thousand separators like german 1.856 kcal instead of 1,856 kcal
* central european figures with a "," as decimal separator cause 102.3 is not right here
* long term: user CONFIGUREABLE DECIMALS in bodymiscale.yaml

**2. Aligned lines for Status (upper left)**
For the left upper status we should make sure that the users gets just 1 icon (scale or warning, but never both cause they are contradicting) in case things are going wrong like for example in a measurement.
181B owners can get longer and misaligned error messages that go over 2 lines where the 2nd line starts
even below that warning icon instead of being aligned to the first line.
![image](https://user-images.githubusercontent.com/20662430/122782583-219e5100-d2b1-11eb-87ce-00f17f2ae166.png)

**Comment by dckiller51** [comment]
Great work. Let me move your post to the card github so we have a better chance of getting other opinions and help on the card.
https://github.com/dckiller51/lovelace-body-miscale-card/issues/2
